homeassistant/components/london_underground/sensor.py

```diff
@@ -55,24 +65,43 @@ async def async_setup_platform(
     session = async_get_clientsession(hass)
 
     data = TubeData(session)
-    await data.update()
+    coordinator = LondonTubeCoordinator(hass, data)
+
+    await coordinator.async_config_entry_first_refresh()
```

Comment:
This method should only be used when setting up a config entry. So we can't use it here.

```suggestion
    await coordinator.async_refresh()
    
    if not coordinator.last_update_success:
        raise PlatformNotReady
```